clients/src/main/java/org/apache/kafka/clients/admin/Admin.java

```diff
@@ -1382,6 +1382,23 @@ default DescribeFeaturesResult describeFeatures() {
         return describeFeatures(new DescribeFeaturesOptions());
     }
 
+    /**
+     * Describe the state of the raft quorum
+     * <p>
+     * The following exceptions can be anticipated when calling {@code get()} on the futures obtained from
+     * the returned {@code DescribeQuorumResult}:
+     * <ul>
+     *   <li>{@link org.apache.kafka.common.errors.ClusterAuthorizationException}
+     *   If the authenticated user didn't have {@code DESCRIBE} access to the cluster.</li>
+     *   <li>{@link org.apache.kafka.common.errors.TimeoutException}
+     *   If the request timed out before the controller could list the cluster links.</li>
+     * </ul>
+     *
+     * @param options The options to use when describing the quorum.
+     * @return The DescribeQuorumResult.
+     */
+    DescribeQuorumResult describeQuorum(DescribeQuorumOptions options);
```

Comment:
By the way, usually these APIs have a default option which leaves out the `XOptions` class. For example, see `describeFeatures()` above?

Also nit: can we not split up the `describeFeatures` APIs?